README.md

```diff
@@ -33,26 +33,26 @@ For Alluxio versions below 1.5.0, use the `alluxio-core-client` artifact.
 <dependency>
   <groupId>org.alluxio</groupId>
   <artifactId>alluxio-core-client-fs</artifactId>
-  <version>1.5.0</version>
+  <version>1.4.0</version>
```

Comment:
@calvinjia This should always be less than 1.5.0, right?

Reply:
Correct

Reply:
got it, done